### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, i added the logic needed to make the Calculator component that i developed in the previous project actually work.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,40 +1,76 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends Component {
```
Comment (DeliceLydia) [inline src/components/Calculator.js:4]: - [x] The Calculator component should import the `calculate.js & operate.js` files from the logic directory currently, you imported `calculate` only please work on that

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,40 +1,76 @@
...
           <div className="conSecond">
-            <button className="colorW" type="button">7</button>
-            <button className="colorW" type="button">8</button>
-            <button className="colorW" type="button">9</button>
-            <button className="colorS" type="button">X</button>
+            <button className="colorW" type="button" name="7" onClick={(e) => { this.getInf(this.state, e.target.name); }}>7</button>
+            <button className="colorW" type="button" name="8" onClick={(e) => { this.getInf(this.state, e.target.name); }}>8</button>
+            <button className="colorW" type="button" name="9" onClick={(e) => { this.getInf(this.state, e.target.name); }}>9</button>
+            <button className="colorS" type="button" name="x" onClick={(e) => { this.getInf(this.state, e.target.name); }}>x</button>
           </div>
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [x] Check how the given design looks like and try to improve
Design

design

yours

sc

- In the calculator, you know that there is always a default `zero` in the screen when you are not doing any operation just add that
- I recognized that after performing an operation let us say `20+20` the response comes with the operator again like `40+` I recommend fixing that It doesn't look nice.. and add a simple space between the operator and the second number

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @denisdiaconu👏,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
Hello!
- The operate.js is already imported in /logic/calculate.js 
- I improved the design of the web app.
- I added the default zero.
- The response comes with an operator so the user can do more calculations.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @denisdiaconu  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**